tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -8280,20 +8173,16 @@ node_modules/chrome-devtools-frontend/front_end/persistence/EditFileSystemView.j
 node_modules/chrome-devtools-frontend/front_end/persistence/EditFileSystemView.js(285,26): error TS2339: Property 'createChild' does not exist on type 'Element'.
 node_modules/chrome-devtools-frontend/front_end/persistence/FileSystemMapping.js(65,22): error TS2694: Namespace 'Common' has no exported member 'Event'.
 node_modules/chrome-devtools-frontend/front_end/persistence/FileSystemMapping.js(73,22): error TS2694: Namespace 'Common' has no exported member 'Event'.
+node_modules/chrome-devtools-frontend/front_end/persistence/FileSystemMapping.js(180,46): error TS2339: Property 'remove' does not exist on type 'Entry[]'.
```

Comment:
we are no longer ignoring the type annotation on

```js
    /** @type {!Object.<string, !Array.<!Persistence.FileSystemMapping.Entry>>} */
    this._fileSystemMappings = {};
```

so we know that Array has no method remove. This is a good error.